plugin/trino-iceberg/pom.xml

```diff
@@ -75,12 +75,6 @@
             <artifactId>trino-orc</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>io.trino.hadoop</groupId>
-            <artifactId>hadoop-apache</artifactId>
-            <scope>compile</scope>
```

Comment:
I'm assuming you removed the scope on purpose?

Reply:
Added it back.